lib/Jit/LLILCJit.cpp

```diff
@@ -290,6 +291,18 @@ CorJitResult LLILCJit::compileMethod(ICorJitInfo *JitInfo,
 
         Passes.add(createRewriteStatepointsForGCPass(false));
         Passes.run(*M);
+
+        // PlaceSafepoints phase inlines the contents of gc.safepointPoll()
+        // at all polling locations. So, once the phase has run, remove the
+        // @gc.safepointPoll() function from the module.
+        Function *SafepointPoll = Context.SafepointPoll;
+#if !defined(NDEBUG)
```

Comment:
@russellhadley: I added a check to verify that there are indeed no callers for the polling function.